packages/relay/test/report.test.ts

```diff
@@ -166,12 +167,30 @@ describe('POST /api/report', function () {
                 expect(res.body).to.have.property('reportId')
             })
 
-        // Verify that the post status is updated
-        await express
-            .get(`/api/post/0?status=${PostStatus.REPORTED}`)
-            .then((res) => {
-                expect(res).to.have.status(200)
-            })
+        // Verify that the post status is updated and content is filtered
+        const afterReportResponse = await express.get(
+            `/api/post/${postId}?status=${PostStatus.REPORTED}`
+        )
+        expect(afterReportResponse).to.have.status(200)
+        expect(afterReportResponse.body).to.not.have.property('content')
+        expect(afterReportResponse.body).to.have.property(
+            'status',
+            PostStatus.REPORTED
+        )
+
+        // Verify that other properties are still present
+        expect(afterReportResponse.body).to.have.property('postId')
+        expect(afterReportResponse.body).to.have.property('publishedAt')
+        expect(afterReportResponse.body).to.have.property('epochKey')
+        // Verify that the post is still accessible but filtered when fetching all posts
+        const allPostsResponse = await express.get('/api/post')
```

Comment:
should call `await postService.updateOrder(db)` to update the post cache. otherwise, you won't get the latest posts.

so, there's another place need to modify.  In this api `POST '/api/report'`, once the report is created, the status of reported object(post, comment) is updated, we need to call `await postService.updateOrder(db)` to refresh the post cache.